app/views/medicaid/insurance_current/edit.html.erb

```diff
@@ -5,7 +5,6 @@
     <div class="form-card__title">
       Are you currently enrolled in a health insurance plan?
     </div>
-
     <p class="text--help text--centered">
```

Comment:
add `form-card__content` ?

Reply:
@jessieay I've been adding the `form-card__content` it to views that have additional content outside of the header (e.g., views with inputs, radios, etc)

Reply:
below there is ` <%= render 'shared/yes_no_form', step: @step, field: :insured %>` which is what has the yes and no buttons -- was assuming you wanted those wrapped. you don't?

Reply:
No, I don't think I want the footer wrapped in form-card__content. 

The patterns that I was thinking:
`form-card__header` --> Header/help text
`form-card__content` --> Any content/form in card.
`form-card__footer` --> Yes/No or Next

Does that makes sense? Or do you think footer should be inside of content?